### DISCUSSION TITLE
非常拜托可以出一个教程

### DISCUSSION THREAD

**Discussion by 905789098** [NateScarlet/auto-derby#208] [Q&A] [answered]
我这边自行磨了差不多三天了，结果下载是下了下来，但任然不知道怎么使用，另外问一下，打开后怎么开始

**Comment by NateScarlet** [comment, marked as answer]
看 README
不考虑出教程 小白用户对项目无用
[推荐阅读](https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md)
